chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,709 +1,806 @@
+									"        \"city\": \"string\",",
+									"        \"country\": \"string\",",
+									"        \"line1\": \"string\",",
+									"        \"line2\": \"string\",",
+									"        \"state\": \"string\",",
+									"        \"zipcode\": \"string\"",
+									"      },",
+									"      \"contact\": {",
+									"        \"contactId\": 0,",
+									"        \"email\": \"string\",",
+									"        \"fullName\": \"string\",",
+									"        \"phoneNumber\": \"string\",",
+									"        \"title\": \"string\"",
+									"      },",
+									"      \"deleted\": false,",
+									"      \"developerCode\": \"string\",",
+									"      \"developerId\": 0,",
+									"      \"lastModifiedDate\": \"string\",",
+									"      \"name\": \"string\",",
+									"      \"selfDeveloper\": false,",
+									"      \"status\": {",
+									"        \"id\": 0,",
+									"        \"status\": \"string\"",
+									"      },",
+									"      \"statusEvents\": [",
+									"        {",
+									"          \"developerId\": 0,",
+									"          \"id\": 0,",
+									"          \"reason\": \"string\",",
+									"          \"status\": {",
+									"            \"id\": 0,",
+									"            \"status\": \"string\"",
+									"          },",
+									"          \"statusDate\": \"2020-10-09T17:08:51.022Z\"",
+									"        }",
+									"      ],",
+									"      \"transparencyAttestations\": [",
+									"        {",
+									"          \"acbId\": 0,",
+									"          \"acbName\": \"string\",",
+									"          \"attestation\": {",
+									"            \"removed\": false,",
+									"            \"transparencyAttestation\": \"string\"",
+									"          }",
+									"        }",
+									"      ],",
+									"      \"website\": \"string\"",
+									"    }",
+									"  ]",
+									"}",
+									"var jsonResponse = pm.response.json();",
+									"pm.test('developer schema is valid', function() {",
+									"  pm.expect(tv4.validate(jsonResponse, developerSchema)).to.be.true;  ",
+									"});",
+									"",
+									"pm.test(\"developer contact details should not include friendly name\", function () {",
+									"    pm.expect(pm.response.text()).to.not.include(\"friendlyName\");",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-KEY",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/developers/107",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"developers"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PUT /developers - developer edit response validation",
```

Comment:
Why is this test in Data driven test folder?
I dont see any developerId being used in this request, if this is right then it should be in non-data driven test

Reply:
I think it got placed in there during merge conflict resolution at some point and wasn't moved after that (it's not being newly added in this PR, shows in diff due to new import rearrangement... I guess). I am going to push a commit to update endpoints, I'll have this moved)

Reply:
pushed a commit to move this one and a couple of others under non-data driven tests